src/main/scala/org/kidsfirstdrc/dwh/variantDb/json/VariantsToJsonJob.scala

```diff
@@ -260,5 +246,20 @@ object VariantsToJsonJob {
               col("phyloP17way_primate_rankscore") as "phylo_p17way_primate_rankscore")
         )
     }
+
+    def withGenes(genes: DataFrame)(implicit spark: SparkSession): DataFrame = {
+      val genesWithoutChromosome = genes.drop("genes_chromosome")
+
+      df
+        .join(genes, col("chromosome") === col("genes_chromosome") && array_contains(df("symbols"), genes("symbol")), "left")
```

Comment:
Chromosome is not required for joining genes. Maybe it's useful for bettter performnance?